task.go

```diff
@@ -55,6 +56,7 @@ type CheckpointTaskOpts func(*CheckpointTaskInfo) error
 
 type TaskInfo struct {
 	Checkpoint *types.Descriptor
+	RootMounts []mount.Mount
```

Comment:
There might be some other changes that make that confusing. Especially since we've moved away from being mostly mount oriented.